tests/lib/config-array/config-dependency.js

```diff
@@ -68,7 +68,8 @@ describe("ConfigDependency", () => {
                     filePath: "filePath?",
                     id: "id?",
                     importerName: "importerName?",
-                    importerPath: "importerPath?"
+                    importerPath: "importerPath?",
+                    original: null
```

Comment:
I think this property shouldn't actually be here. It seems that the intent was to avoid serializing plugins (and other loaded dependencies). Like with `definition`, I believe we should hide `original` as well:

https://github.com/eslint/eslintrc/blob/af564ff562bd4f8c40163b8a44ed9d4989adff6a/lib/config-array/config-dependency.js#L102-L109

Reply:
Ah, that makes sense. I'll update that.